win32ss/user/user32/windows/defwnd.c

```diff
@@ -390,11 +390,11 @@ User32DefWindowProc(HWND hWnd,
             {
                 if (bUnicode)
                 {
-                    SendMessageW(GetParent(hWnd), Msg, wParam, lParam);
+                    SendMessageW(GetParent(hWnd), Msg, (WPARAM)hWnd, lParam);
                 }
                 else
                 {
-                    SendMessageA(GetParent(hWnd), WM_CONTEXTMENU, wParam, lParam);
+                    SendMessageA(GetParent(hWnd), WM_CONTEXTMENU, (WPARAM)hWnd, lParam);
```

Comment:
Here, the second parameter could be "`Msg`" as in the unicode case.

Reply:
sure